src/app/zap-templates/templates/app/im-cluster-command-handler.zapt

```diff
@@ -37,7 +37,12 @@ Protocols::InteractionModel::Status DispatchServerCommand(CommandHandler * apCom
     {{/first}}
         case Commands::{{asUpperCamelCase commandName}}::Id: {
             Commands::{{asUpperCamelCase commandName}}::DecodableType commandData;
+
```

Comment:
```suggestion
```
I see no reason for the whitespace churn there...

Reply:
Woops, apologies, I looked at the output of cluster-objects-src.zapt so carefully but forgot to double-check this one